invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -31,15 +31,14 @@ class DublinCoreSchema(BaseSchema):
     descriptions = fields.Method("get_descriptions")
     publishers = fields.Method("get_publishers")
     types = fields.Method("get_types")
-    sources = fields.Method("get_sources")
+    # TODO: sources = fields.List(fields.Str(), attribute="metadata.references")
+    sources = fields.Constant(
```

Comment:
why do we have this as TODO? what is missing?

Reply:
That was there before the code changes. i.e. not part of this PR, I just made it more clear.

IMO, I think the [`references`](https://github.com/inveniosoftware/invenio-rdm-records/blob/a02775d565795a2574bd6d8812143b64917de6fd/invenio_rdm_records/services/schemas/metadata.py#L367) is just not used at the moment (i.e. not exposed in the UI). We are defaulting to missing (as it was before, this behaviour didn't change) to avoid data migration afterwards, so right now is just ignored.